interface/src/app/tribe_client/genesets.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.tribe_client.genesets', [
+  'adage.tribe_client.resource'
+])
+
+// Geneset Search Factory
+// Allows for retreiving for and interacting with search results for genesets.
+.factory('GenesetSearch', ['Genesets', function(Genesets) {
+  var genesets = [];
```

Comment:
Hah. I did indeed just now see your "it lives!" message. In part, GitHub was being sluggish to show your replies. The other part is stepping away for the afternoon of kid activities I mentioned just before I dashed off this "wait a second..." note.

Glad you've sorted it out! I'm looking over what you did now.